presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/AddIntermediateAggregations.java

```diff
@@ -150,7 +153,7 @@ public Result apply(AggregationNode aggregation, Captures captures, Context cont
         return Optional.of(node.replaceChildren(builder.build()));
     }
 
-    private PlanNode addGatheringIntermediate(AggregationNode aggregation, PlanNodeIdAllocator idAllocator)
+    private PlanNode addGatheringIntermediate(AggregationNode aggregation, PlanNodeIdAllocator idAllocator, SymbolAllocator symbolAllocator)
```

Comment:
Previous comment is not addressed (https://github.com/prestodb/presto/pull/12710#discussion_r284573338)